scm/github/webhook.go

```diff
@@ -437,8 +437,8 @@ func (c *client) processIssueCommentEvent(h *library.Hook, payload *github.Issue
 		fmt.Sprintf("https://%s/%s/settings/hooks", h.GetHost(), payload.GetRepo().GetFullName()),
 	)
 
-	// skip if the comment action is deleted
-	if strings.EqualFold(payload.GetAction(), "deleted") {
+	// skip if the comment action is deleted or not part of a pull request
+	if strings.EqualFold(payload.GetAction(), "deleted") || !payload.GetIssue().IsPullRequest() {
```

Comment:
this tweak would make this PR a breaking change, right?

Reply:
I was looking at this... this actually just makes the return nicer. If you make a comment on an issue, this is the current return: 
```json
{
  "error":  "unable to process webhook: failed to create pipeline for EastonCrupper/myvela: empty pipeline commit provided"
}
```

Now it will be `no build to process`. Looks like ever since pipelines, the `comment` event got a little wonky outside PR contexts.

Reply:
makes sense. i just quick scanned the docs and we do only talk about comments in the context of PRs as far as I can tell, so i think we're good. thanks